static/accounts/style.css

```diff
@@ -315,4 +315,20 @@ input {
 	.radio-input-label1{
 		margin-right: 20px;
 	}
+}
+
+#password_div{
+	display: table;
+	width: 100%;
+	margin-left: -12px;
+}
+
+#password{
+	display: inline-block;
```

Comment:
I don't see the need for this property as all input fields are already `inline-block` by default.